geode-redis/src/main/java/org/apache/geode/redis/internal/executor/server/InfoExecutor.java

```diff
@@ -58,6 +61,9 @@ private String getSpecifiedSection(ExecutionHandlerContext context,
       case "persistence":
         result = getPersistenceSection();
         break;
+      case "replication":
+        result = getReplicationSection();
+        break;
```

Comment:
Seems like we also need to add the following cases: `clients, memory, stats, and keyspace`

Reply:
story made for this

Reply:
story added to implement this.